src/implementations/twig/components/button/button.story.js

```diff
@@ -1,7 +1,5 @@
-import { withKnobs, text, select, boolean } from '@storybook/addon-knobs';
```

Comment:
Putting the request here as related to the whole file: let's please disable the knobs as a first step of the migration as discussed in https://github.com/ec-europa/europa-component-library/pull/1820#discussion_r540850867